controllers/front/ProductController.php

```diff
@@ -318,6 +323,14 @@ public function displayAjaxQuickview()
     public function displayAjaxRefresh()
     {
         $product_for_template = $this->getTemplateVarProduct();
+        $product_attribute_id = $product_for_template['id_product_attribute'];
+
+        if ($this->combinations) {
+            $mininal_product_combination_quantity = intval($this->combinations[$product_attribute_id]['minimal_quantity']);
```

Comment:
A little typo here. `mininal`? ^^
Also, for the new variable name, we should use camelCase